projects/components/src/table/table.component.ts

```diff
@@ -330,6 +343,85 @@ export class TableComponent
     this.rowStateSubject.complete();
     this.columnStateSubject.complete();
     this.columnConfigsSubject.complete();
+
+    this.mouseMoveFunction && this.mouseMoveFunction();
+    this.mouseUpFunction && this.mouseUpFunction();
+  }
+
+  public trackItem(_index: number, column: TableColumnConfigExtended): string {
+    return `${column.id}-${column.width}`;
+  }
+
+  public queryHeaderRowElement(): HTMLElement {
+    return this.elementRef.nativeElement.querySelector('cdk-header-row');
+  }
+
+  public queryHeaderCellElement(index: number): HTMLElement {
+    return this.elementRef.nativeElement.querySelectorAll('cdk-header-cell').item(index);
+  }
+
+  private getVisibleColumnConfig(index: number): TableColumnConfigExtended {
```

Comment:
Is this assuming the index is from visible cols rather than all cols? If so, we should rename it, because looks like a bug otherwise.

Reply:
The method name should be clear enough.